app/views/families/_form.html.erb

```diff
@@ -1,14 +1,54 @@
-<div class="container">
+<table class="form-table">
```

Comment:
Please don't use a table to layout a form. It is not semantically correct.
Table should be used for tabular data, not for laying out the page.

Happy to help you with styling form fields...

Reply:
I changed all the tables to be div with lists and I adapted the scss. All in sha: `3361eb6`